src/WebOptimizer.Less.csproj

```diff
@@ -5,7 +5,7 @@
         <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\WebOptimizer.LESS.xml</DocumentationFile>
         <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
         <PackageTags>LESS, bundle, minify, css, asp.net core</PackageTags>
-        <Version>1.0.0-beta</Version>
+        <Version>2.7.2-beta</Version>
```

Comment:
This shouldn't be updated. The build server handles the versioning automatically